app/src/main/java/com/github/steroidteam/todolist/model/todo/Tag.java

```diff
@@ -0,0 +1,76 @@
+package com.github.steroidteam.todolist.model.todo;
+
+import android.graphics.Color;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+/**
+ * A class that represents a tag. Tags contain a body and a color. They are used to classify to-do
+ * lists in different categories.
+ */
+public class Tag {
+    UUID uuid;
+    private String body;
+    private int color;
+    private Set<UUID> listsIds;
+    // TODO : Move to database when adding persistence
+    public static Set<Tag> tags = new HashSet<>();
+
+    public Tag(String body, int color, UUID listId) {
+        this.uuid = UUID.randomUUID();
+        this.body = body;
+        this.color = color;
+        listsIds = new HashSet<>();
+        listsIds.add(listId);
+        tags.add(this);
+    }
+
+    public Tag(String body, UUID listId) {
+        this(body, Color.WHITE, listId);
```

Comment:
It'd be cool if the color were randomly generated, but this is just nitpicking. We can add the colour picker or whatever view we use for colours.

Reply:
In fact, do we even use this constructor?